tests/FormBuilderTest.php

```diff
@@ -172,6 +172,27 @@ public function testRenderCheckboxWithOldInput()
 		$this->assertEquals($expected, $result);
 	}
 
+	public function testRenderCheckboxArrayWithOldInput()
+	{
+		$oldInput = Mockery::mock('AdamWathan\Form\OldInput\OldInputInterface');
+		$oldInput->shouldReceive('hasOldInput')->andReturn(true);
+		$oldInput->shouldReceive('getOldInput')->with('favourite_foods')->andReturn(array(0 => 'fish', 1 => 'chips'));
```

Comment:
Can use 5.4 array syntax here too, and I don't think we need to specify the keys because it doesn't look like their values are important anyways. If the 0 and 1 had meaning elsewhere I'd say keep em but doesn't seem to be the case.